scripts/openstack.py

```diff
@@ -0,0 +1,156 @@
+import argparse
```

Comment:
The inability of docopt to issue a useful error message when the wrong option is being used makes it extremely painful to use, on a daily basis. (yes, today, once more I faced a cryptic usage display because I repeated an option twice by mistake...).